sentences/de/homeassistant_HassGetCurrentTime.yaml

```
@@ -0,0 +1,8 @@
+language: de
+intents:
+  HassGetCurrentTime:
+    data:
+      - sentences:
+          - "wie spät ist es"
+          - "wie viel uhr (ist es[ jetzt]|haben wir)"
```

```suggestion
          - "wie viel uhr (ist es|haben wir)[ jetzt]"
```
I think in some areas "wie viel uhr haben wir jetzt" is also a valid question.